rocketmad/utils.py

```diff
@@ -1242,12 +1263,34 @@ def parse_geofence_file(geofence_file):
                     log.debug('Found geofence: %s.', name)
                 else:  # Coordinate line.
                     lat, lon = line.split(",")
-                    LatLon = {'lat': float(lat), 'lon': float(lon)}
-                    geofences[-1]['polygon'].append(LatLon)
+                    coord = (float(lat), float(lon))
+                    geofences[-1]['polygon'].append(coord)
 
     return geofences
 
 
+def get_geofence_box(geofence):
+    sw_lat = 100
```

Comment:
please use 90/180 here, not 100/200 that first reaction is 'wtf is this?' :)

Reply:
Guess we don't have anybody mapping on those latitudes 😂